manifest.md

```diff
@@ -139,7 +139,7 @@ Unlike the [Manifest List](#manifest-list), which contains information about a s
 
 - **`layers`** *array*
 
-    Each item in the array MUST be a [descriptor](descriptor.md).
+    This property MAY be empty, an image can have an empty layers array. Each item in the array MUST be a [descriptor](descriptor.md).
     The array MUST have the base image at index 0.
     Subsequent layers MUST then follow in the order in which they are to be layered on top of each other.
```

Comment:
These two lines should be restructured to avoid requiring a zero-index layer.  Something like:

    The root filesystem MUST match an empty directory onto which the layers have been [applied](layer.md#applying) in the listed order.

With “match” instead of “be” to allow folks to apply via a union filesystem or whatever.  As long as the finished product is right.